client/src/pages/Prospects.js

```diff
@@ -209,14 +201,30 @@ const Prospects = () => {
     handleSelectedProspects(newSelected);
   };
 
-  let listOfProspects = data;
+  // Filter inported_from's option
+  const filteredImportedFrom = importedFromTerm.name === ''
+  ? data 
+  : data.filter(data => {
+    return data['Imported_from'] === importedFromTerm.name;
+  })
+ 
+  // Filter status's option
+  const filteredStatus = statusTerm.name === ''
+  ? filteredImportedFrom
+  : filteredImportedFrom.filter(data => {
+    return data['Status'] === statusTerm.name;
+  })
```

Comment:
Nice use of the filter function, btw if you don't add a return statement, will it still work?

Reply:
Yes! If the code is more than one line only it will need the return statement.